modules/nf-core/dragmap/hashtable/main.nf

```diff
@@ -2,7 +2,7 @@ process DRAGMAP_HASHTABLE {
     tag "$fasta"
     label 'process_high'
 
-    conda (params.enable_conda ? "bioconda::dragmap=1.2.1" : null)
+    conda "bioconda::dragmap=1.3.0"
```

Comment:
seems like something bad is happening here: diff file says 1.2.1 for conda, but not for the containers